ex02.ex

```diff
@@ -53,8 +55,12 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
-
+  first2equal = fn list -> 
```

Comment:
# -1

In this case, I think it would be clearer to just use a multi-headed function

~~~ elixir

  first2equal = fn
                  [ a, a | _ ] -> true
                  _            -> false
                end
~~~